neo/Implementations/Blockchains/LevelDB/LevelDBBlockchain.cs

```diff
@@ -106,6 +108,18 @@ public LevelDBBlockchain(string path)
             thread_persistence.Start();
         }
 
+        public override Block CurrentBlock
+        {
+            get
+            {
+                if (persisting_block != null)
+                {
+                    return persisting_block;
+                }
+                return GetBlock(Height);
```

Comment:
Maybe in the `else` was good option set `persisting_block=GetBlock (Height)`

Reply:
I want `persisting_block` to be `null` unless execution is in the middle of `LeveldbBlockchain.Persist`